### ISSUE TITLE
Please add a setting of basic inventory hopper

### ISSUE THREAD

**Issue by Plain-Second** [tom5454/Toms-Storage#359]
Please add settings of basic inventory hopper to set its Item transfer speed. Because when I make a relatively efficient machine, and then use this mod to collect and sort, the speed of the basic inventory hopper can't keep up with items, and then the items will be stuck.
    I've tried to modify the file myself, but when I found the item file and opened it, I found it too complicated to start, so I came to ask for help（＾∀＾）

**Comment by Eslorex** [comment]
I need this too. I have the mod in my modpack. I need to transfer tons of items to my inventory network. But that hopper bottlenecks me so bad.

**Comment by Plain-Second** [comment]
My current solution is to get two warehouses, the first one collects items first, and then uses this hopper to slowly sort to the second warehouse

**Comment by Shadowdane** [comment]
Agreed on this the basic inventory hopper is way too slow!    I had 2 of my farms break because the speed of the basic inventory hopper is too slow to keep up with the farm output.   I had to manually nerf the item output to keep things from overflowing the buffer chests I had setup that feed into the storage network.

My suggestion maybe a stack upgrade could slot into it to grab up to 1 stack at the same time.   Or even a speed upgrade to have it pull items every 2 ticks or something.   Or a combo of both.   I know the system can handle dumping items in fast as I've dumped in entire inventories into the console in 1 click and it handles it just fine.

**Comment by Nox-Veneficus** [comment]
any chance on getting this implemented tom! I would hate to have to disable inventory hoppers due to how laggy they are.
